src/techs/techs.service.ts

```diff
@@ -71,6 +79,48 @@ export class TechsService {
         });
     };
 
+    async updateTechStackSelections(
+        req,
+        teamId: number,
+        updateTechSelectionsDto: UpdateTechSelectionsDto,
+    ) {
+        const categories = updateTechSelectionsDto.categories;
+
+        //count selections in categories for exceeding MAX_SELECT_COUNT
+        categories.forEach((category) => {
+            const selectCount = category.techs.reduce(
+                (acc: number, tech) => acc + (tech.isSelected ? 1 : 0),
+                0,
+            );
+            if (selectCount > MAX_SELECTION_COUNT)
+                throw new BadRequestException(
+                    `Only ${MAX_SELECTION_COUNT} selections allowed per category`,
+                );
+        });
+
+        const voyageMemberId = await this.findVoyageMemberId(req, teamId);
+        if (!voyageMemberId)
+            throw new BadRequestException("Invalid User or Team Id");
+
+        //extract techs to an array for .map
+        const techsArray: any[] = [];
+        categories.forEach((category) => {
+            category.techs.forEach((tech) => techsArray.push(tech));
+        });
+        return this.prisma.$transaction(
+            techsArray.map((tech) => {
+                return this.prisma.teamTechStackItem.update({
+                    where: {
+                        id: tech.techId,
+                    },
+                    data: {
+                        isSelected: tech.isSelected,
```

Comment:
I checkout the branch locally and ran `yarn start:dev` that gave me an error saying "error TS2353: Object literal may only specify known properties, and 'isSelected' does not exist in type '(Without<TeamTechStackItemUpdateInput, TeamTechStackItemUncheckedUpdateInput> & TeamTechStackItemUncheckedUpdateInput) | (Without<...> & TeamTechStackItemUpdateInput)'."
on line 117 techs.service.ts. 

was there any changes done to the prisma schema?

Reply:
Yes - it was the first change I made and I forgot to mention it!  `yarn migrate` is needed to update